interface/ceed-operator.c

```diff
@@ -905,6 +1023,16 @@ int CeedOperatorDestroy(CeedOperator *op) {
   ierr = CeedQFunctionDestroy(&(*op)->dqf); CeedChk(ierr);
   ierr = CeedQFunctionDestroy(&(*op)->dqfT); CeedChk(ierr);
 
+  // Destroy fallback
+  if ((*op)->fallback) {
+    ierr = (*op)->qffallback->Destroy((*op)->qffallback); CeedChk(ierr);
+    ierr = CeedFree(&(*op)->qffallback); CeedChk(ierr);
+    ierr = (*op)->fallback->Destroy((*op)->fallback); CeedChk(ierr);
```

Comment:
We can't we `CeedOperatorDestroy(&(*op)->fallback);`?

Reply:
The fallback contains references to memory that has been freed, so we need to only call the backend destructor code.